GitUI/UserControls/RevisionGrid/Columns/BuildStatusColumnProvider.cs

```diff
@@ -79,36 +78,10 @@ public override void OnCellPainting(DataGridViewCellPaintingEventArgs e, GitRevi
                 return;
             }
 
-            Size size;
+            string text = (AppSettings.ShowBuildStatusIconColumn ? revision.BuildStatus.StatusIcon : string.Empty)
```

Comment:
I was confused about this change, expected this to be handled as two separate columns, but it only seem to be so in the settings

```suggestion
            // Settings presents this as two columns but the variance is in the text
            string text = (AppSettings.ShowBuildStatusIconColumn ? revision.BuildStatus.StatusIcon : string.Empty)
```